proto/greptime/v1/region/server.proto

```diff
@@ -187,18 +187,12 @@ message RegionColumnDef {
 
 // Request of bulk ingestion API.
 message BulkInsertRequest {
+  uint64 region_id = 1;
   oneof body {
-    ArrowIpc arrow_ipc = 1;
+    ArrowIpc arrow_ipc = 2;
   }
```

Comment:
This may be a breaking change in the API level. We may mark it as breaking in this PR and https://github.com/GreptimeTeam/greptimedb/pull/6178.